WHITEPAPER.md

```diff
@@ -195,3 +195,17 @@ The following phases may or may not be useful. I think it would be valuable if w
 * [The Simian Army](https://github.com/Netflix/SimianArmy) - A suite of tools for keeping your cloud operating in top form.
 * [Toxiproxy](https://github.com/Shopify/toxiproxy) - A TCP proxy to simulate network and system conditions for chaos and resiliency testing.
 * [Wiremock](http://wiremock.org/) - API mocking (Service Virtualization) which enables modeling real world faults and delays
+
+
+## Appendix A: Additional Material
+
+- [Principles of Chaos Engineering](http://principlesofchaos.org/)
+
+- [Chaos Engineering: Building Confidence in System Behavior through Experiments](https://www.oreilly.com/webops-perf/free/chaos-engineering.csp)
```

Comment:
It's missing the author and book reference.
By the way, the whole book seems to be available here for free (no signup required) https://www.oreilly.com/ideas/chaos-engineering